### PR TITLE
Yamls for JPSSRR sea-ice obs in cycling test

### PR DESCRIPTION
**Pull request by apchoiCMD**
#### This PR is to add the yaml keys
- Yaml keys for JPSSRR sea-ice for the cycling
- The file names of JPSSRR type sea-ice are the same in `NH` and `SH`, please see in `yamls` and `obsprep_config.yaml`
- The preliminary results here/below
GFSv17-3DVAR-C384mx025_gdasprepoceanobs_202106300600_NH
- Obs for the SH is not ready yet

Related to https://github.com/NOAA-EMC/GDASApp/pull/1259

### CODE REVIEW

File: parm/soca/obs/config/icec_viirs_n20_l2_north.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+obs filters:
+- filter: Domain Check
+  where:
+  - variable: {name: GeoVaLs/sea_area_fraction}
+    minvalue: 0.9
+- filter: Bounds Check
+  minvalue: 0.0
+  maxvalue: 1.0
+#- filter: Background Check
+#  threshold: 5.0
```
Comment (JohnSteffen-NOAA) [inline parm/soca/obs/config/icec_viirs_n20_l2_north.yaml:30]: Do we need to keep this background check filter commented out? If not, then delete.

File: parm/soca/obs/config/icec_viirs_n20_l2_south.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+obs filters:
+- filter: Domain Check
+  where:
+  - variable: {name: GeoVaLs/sea_area_fraction}
+    minvalue: 0.9
+- filter: Bounds Check
+  minvalue: 0.0
+  maxvalue: 1.0
+#- filter: Background Check
+#  threshold: 5.0
```
Comment (JohnSteffen-NOAA) [inline parm/soca/obs/config/icec_viirs_n20_l2_south.yaml:30]: Same comment as the *north.yaml

File: parm/soca/obs/config/icec_viirs_n20_l2_south.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- filter: Domain Check
```
Comment (JohnSteffen-NOAA) [inline parm/soca/obs/config/icec_viirs_n20_l2_south.yaml:31]: Why is there a domain check for sea_surface_temperature > 2.0 and also > 0? Isn't one sufficient?

File: parm/soca/obs/config/icec_viirs_n20_l2_north.yaml (hunk 1 above)
Comment (apchoiCMD) [reply parm/soca/obs/config/icec_viirs_n20_l2_north.yaml:30]: Good point but now as same as what `amsr2` has

File: parm/soca/obs/config/icec_viirs_n20_l2_south.yaml (hunk 2 above)
Comment (apchoiCMD) [reply parm/soca/obs/config/icec_viirs_n20_l2_south.yaml:30]: the same answer above

File: parm/soca/obs/obs_list.yaml [hunk 4]
```diff
@@ -15,6 +15,8 @@ observers:
 # Ice concentration
 - !INC ${MARINE_OBS_YAML_DIR}/icec_amsr2_north.yaml
 - !INC ${MARINE_OBS_YAML_DIR}/icec_amsr2_south.yaml
+#- !INC ${MARINE_OBS_YAML_DIR}/icec_viirs_n20_l2_north.yaml
```
Comment (KatherineLukens-NOAA) [inline parm/soca/obs/obs_list.yaml:18]: Did you mean to comment this out? Same question for "south"

Comment (apchoiCMD) [reply parm/soca/obs/obs_list.yaml:18]: Not part of `CI` yet, we will need to comment out when we have to use them-

**Comment by guillaumevernieres** [comment]
Does the figure above has all the obs for the cycle? If so, there's probably a bug somewhere @apchoiCMD .

**Review by AndrewEichmann-NOAA — COMMENTED** [review]
Nothing to add

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Comment by apchoiCMD** [comment]
> Does the figure above has all the obs for the cycle? If so, there's probably a bug somewhere @apchoiCMD .

GFSv17-3DVAR-C384mx025_gdasprepoceanobs_20240618 00Z_SH

From the original files, zero values appears-

**Pull request merged**